### PR TITLE
Fix docstring errors in `__init__.py`

### PR DESCRIPTION
**Pull request by timcallow**
Fixes the warnings:

docerrors_screenshot

### CODE REVIEW

File: atoMEC/__init__.py [hunk 1]
```diff
@@ -179,6 +169,7 @@ def radius(self, radius):
 
     @property
     def density(self):
+        """float: the mass density of the material in units :math:`\mathrm{g\ cm}^{-3}`"""
```
Comment (DanielKotik) [inline atoMEC/__init__.py:172]: Unit does not get rendered correctly. Turning string into a raw string fixes it.
```suggestion
        r"""float: the mass density of the material in units :math:`\mathrm{g\ cm}^{-3}`"""
```

File: atoMEC/__init__.py [hunk 2]
```diff
@@ -168,6 +156,8 @@ def units_density(self, units_density):
 
     @property
     def radius(self):
+        r"""float: radius of the Wigner-Seitz sphere, defined as :math:`0.5*a_i`,
```
Comment (DanielKotik) [inline atoMEC/__init__.py:159]: Either omit the `*` symbol or rewrite equation to
```suggestion
        r"""float: radius of the Wigner-Seitz sphere, defined as :math:`a_i /2`,
```

**Review by DanielKotik — APPROVED** [review]
Great! Fixing the two small issues, and the PR can be merged.:)

**Review by timcallow — COMMENTED** [review]
Good finds @DanielKotik

**Pull request merged**